versioned_docs/version-2.7/reference-guides/rancher-webhook.md

```diff
@@ -34,11 +34,11 @@ It provides essential protection for Rancher-managed clusters, preventing securi
 
 ## What Resources Does the Webhook Validate?
 
-An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).
+You can find an in-progress list of the resources that the webhook validates in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). These docs are organized by group/version and resource (top-level header is group/version, next level header is resource). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have this file).
```

Comment:
The changes in this file don't seem to be synced to the main page in `docs/reference-guides/rancher-webhook.md`. Would you mind copying the changes over to the main file as well?